src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -206,6 +207,7 @@ public class BssEpisode
     public long nhs_number { get; set; }
     public string? episode_type { get; set; }
     public DateTime change_db_date_time { get; set; }
+    [Format("dd/MM/yyyy")]
```

Comment:
Can you communicate this fix with Chris but remove it from this PR since we have a separate ticket for it on the board now, please

Reply:
Ready for review